packages/babel-generator/test/index.js

```diff
@@ -847,7 +847,7 @@ suites.forEach(function (testSuite) {
             } else {
               const result = run();
 
-              if (options.sourceMaps) {
+              if (options.sourceMaps || options.sourceMap) {
```

Comment:
Nit: the `sourceMap` option is synonym of `sourceMaps`. Since we [recommend `sourceMaps` option](https://babeljs.io/docs/en/options#sourcemap) in the doc, can you replace the test options using `sourceMap`?

Reply:
For some reason another place goes wrong when I use `sourceMaps`, so I've left it out for the sake of convenience, but since you mentioned this, I'll look at it later